src/ert/gui/main.py

```diff
@@ -152,6 +176,41 @@ def _setup_suggester(suggestions, ert_window=None):
     text = "\n".join(suggestions)
     lines.setPlainText(text)
 
+    help_button_frame = QFrame()
+    help_button_frame.setFrameStyle(QFrame.StyledPanel)
+    button_layout = QHBoxLayout()
+    help_button_frame.setLayout(button_layout)
+
+    button_size = QSize(-1, -1)
+    helpbuttons = []
+
+    pm = ErtPluginManager()
+    help_links = pm.get_help_links()
+
+    for menu_label, link in help_links.items():
+        button = QPushButton(menu_label)
+        button.clicked.connect(lambda: _clicked_help_button(menu_label, link))
```

Comment:
This is a bit of a gotcha in python. A lambda closure inside of a for loop has some very special semantics:
https://stackoverflow.com/questions/3431676/creating-functions-or-lambdas-in-a-loop-or-comprehension

Reply:
https://docs.python-guide.org/writing/gotchas/#late-binding-closures

Reply:
Thanks for the heads up on this. I wasn't aware of that. Easy fix.